api_types.ts

```diff
@@ -1013,6 +1004,12 @@ export enum TableRole {
   GroupMembers = 'groupMembers',
 }
 
+export enum IncrementalPermissionCrawlType {
+  RecrawlAllUpdatedItems = 'recrawl_all_updated_items',
+  RecrawlOnlyNewItems = 'recrawl_only_new_items',
```

Comment:
If we're only visiting new items, are we "recrawling" them or just "crawling" them?  This reads a bit confusing to me.